model/src/pyrenew/latent/infection_functions.py

```diff
@@ -6,7 +6,8 @@
 import jax
 import jax.numpy as jnp
 from jax.typing import ArrayLike
-from pyrenew.convolve import new_convolve_scanner, new_double_scanner
+from pyrenew.convolve import new_convolve_scanner, new_double_convolve_scanner
+from pyrenew.transformation import ExpTransform, IdentityTransform
```

Comment:
Yes, now that I think of it, perhaps importing directly from numpyro may be better.

```suggestion
from numpyro.distributions.transforms import ExpTransform, IdentityTransform
```